pkg/pillar/cmd/domainmgr/domainmgr.go

```diff
@@ -2022,6 +2120,56 @@ func xlUnpause(domainName string, domainId int) error {
 	return nil
 }
 
+// Wrapper for domain shutdown thru xlShutdown or rktStop
+func wrapDomainShutdown(status types.DomainStatus, force bool) error {
```

Comment:
wouldn't a more natural name for this function simply be domainShutdown? I mean containers ARE domains in our implementation.

Reply:
Taken care of in latest commit